plugin/Player.pm

```diff
@@ -94,6 +94,11 @@ sub initPrefs {
 
 	# make sure any preferences unique to this client may not have set are set to the default
 	$serverPrefs->client($client)->init($defaultPrefs);
+	
+	$prefs->client($client)->init({
+		syncPower => 1,
+		syncVolume => 1
+	});
```

Comment:
I don't think I can do that, or I have to use another name. 

syncPower and syncVolume already exist in "normal" client prefs and can bet set through the UI, but they mean something different. a client that has syncPower set will be powered on/off (by LMS) if any other client of the group is powered on/off, which is somethign I absolutely want to avoid. 

In the case of a group, what I want is that when you power that special player, then it powers on/off *all* the members. When a group member is powered off, the rest should continue

At least, if we decide to have such prefs being part of client's prefs handle in server.prefs, they should have a different name

Reply:
That‘s prefs in the plugin‘s namespace. That‘s why I had to add this second init call. It will not conflict with the other prefs of the same name (I was confused at first, too)

Reply:
Undertsood - I have to say I'm a bit confused with how to use package prefs other than the very basic things I did before.  I have to look at what client() method does. Its this just setting a prefix/namespace that uses one attribute of its's parameter (a Slim::Player::Client object)

Reply:
prefs are namespaced. There are global prefs (stored in server.prefs, eg music folder) and plugin specific prefs (eg. player name). And any namespace can have per-player prefs as well as server wide prefs. $prefs->client($id) gives you access to the per-player prefs in a namespace.